packages/object-safe-get/index.js

```diff
@@ -22,6 +22,9 @@ module.exports = get;
 */
 
 function get(obj, props) {
+  if (!obj) {
+    return undefined;
```

Comment:
maybe we should return obj instead (so if obj is null it returns null). This is also consistent with the logic for sub-properties